apps/vercel/frontend/src/locations/ConfigScreen.tsx

```diff
@@ -46,19 +48,24 @@ const ConfigScreen = () => {
     };
   }, [parameters, sdk]);
 
+  const vercelClient = new VercelClient(parameters.vercelAccessToken);
```

Comment:
Thoughts on if we want to store this in a state and update it in a useEffect?  That way it's not creating a new vercel client every rerender

Reply:
Good idea George! I will follow-up up with some adjustments in subsequent PRs :)!!